src/plot_styles.py

```diff
@@ -124,6 +133,8 @@ class PlotStylesWindow(Adw.Window):
     minor_tick_width = Gtk.Template.Child()
     major_tick_length = Gtk.Template.Child()
     minor_tick_length = Gtk.Template.Child()
+    set_active_style = Gtk.Template.Child()
```

Comment:
just to keep it consistent, I'd sort these after font_chooser. 8sort in order of visual appearance)